zng/docs/spec.md

```diff
@@ -476,33 +503,33 @@ is an error.
 When errors are encountered parsing ZNG, an implementation should return a
 corresponding error and allow ZNG parsing to proceed if desired.
 
-### 3.1.1 Type Binding
+### 4.1.1 Type Binding
 
 A type binding has the following form:
 ```
 #<type-tag>:<type-string>
 ```
 Here, `<type-tag>` is a string decimal integer and `<type-string>`
-is a string defining a type according to the ZNG type syntax creates a binding
-between the indicated tag and the indicated type.
+is a string defining a type according to the ZNG type syntax, which creates
```

Comment:
The original wording was grammatically odd. After my adjustment, is this text still clearly communicating the original intent?

Reply:
Yes, but it still reads oddly to me. How about this?
```suggestion
is a string defining a type according to the ZNG type syntax. They create
```

Reply:
👍